alpha/lib/model/DeliveryProfileLiveAppleHttp.php

```diff
@@ -120,13 +120,61 @@ protected function isDvrContent($content) {
 	}
 	
 	/**
-	 * Build all streaming flavors array
+	 * Fetch the manifest from the remote Wwoza server and build all flavors array
 	 * @param string $url
 	 */
-	private function buildM3u8Flavors($url, array &$flavors, array $kLiveStreamParamsArray, $flavorBitrateInfo = array())
+	private function buildProxiedFlavorsArray($url, array &$flavors, $doaminPrefix)
 	{
-		$domainPrefix = $this->getDeliveryServerNodeUrl(true);
+		if($this->getDisableExtraAttributes())
+			$url = kDeliveryUtils::addQueryParameter($url, "attributes=off");
+		
+		$manifest = KCurlWrapper::getContent($url);
+		if(!$manifest)
+			return;
```

Comment:
yes maybe that would be a good idea foratter identifying issues that occurred in the past.

I will add the debug.